components/Form/TableFilterForm.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import Form from './Form';
+import FormField from './FormField';
+import SubmitButton from './SubmitButton';
+import ClearButton from './ClearButton';
+import styles from './TableFilterForm.css';
+import * as data from './TableFilter.config';
+import { connect } from 'react-redux';
+import { submitForm, clearForm } from '../../actions';
+
+const TableFilterForm = ({ submitForm, clearForm }) => (
+  <Form name="tableFilter" className={styles.form}>
+    <FormField
+      name="withHeroId"
```

Comment:
shouldn't the name be snake_case to match the API params?  I think you have a conversion function somewhere, but isn't it more obvious to make it match directly?